### PR TITLE
Add IT support for AD kibana plugin

### PR DESCRIPTION
**Pull request by weicongs-amazon**
### Changes ###
Add IT support for AD kibana plugin

### Testing ###
* Deb: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/891775070?check_suite_focus=true
* Tar: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/891983238?check_suite_focus=true
* Docker: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/892636886?check_suite_focus=true
* RPM: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/895454356?check_suite_focus=true
    * failed since the disk size is too small in our hostrunner. It should be fixed in the new runner

### Notes ###
* The IT step fails due to the known issue from AD kibana plugin. It has been fixed in master
* The IT can't run well in windows.  will have a follow up with AD kibana team

### CODE REVIEW

File: .github/workflows/test-build-rpm.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
   #  - cron: '0 10 * * *'
   repository_dispatch:
     types: [test-build-rpm]
-
+
```
Comment (peterzhuamazon) [inline .github/workflows/test-build-rpm.yml:8]: Remove this space

File: .github/workflows/test-build-rpm.yml [hunk 2]
```diff
@@ -340,8 +340,95 @@ jobs:
           cd $GITHUB_WORKSPACE/../k-NN
           ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest
 
-  Cleaning-Host:
+  Test-Kibana-Plugins-Without-Security:
```
Comment (gaiksaya) [inline .github/workflows/test-build-rpm.yml:343]: Do we have with and without security testing in kibana too?

Comment (weicongs-amazon) [reply .github/workflows/test-build-rpm.yml:343]: I think so. Different users might see different UIs

File: .github/workflows/test-build-rpm.yml (hunk 1 above)
Comment (weicongs-amazon) [reply .github/workflows/test-build-rpm.yml]: Fixed

File: .github/workflows/test-build-rpm.yml (hunk 2 above)
Comment (gaiksaya) [reply .github/workflows/test-build-rpm.yml:343]: So do we need to mention it in all job names? Just saw without security here! For all others it is not mentioned whether it is with or without security

Comment (peterzhuamazon) [reply .github/workflows/test-build-rpm.yml:343]: I like to follow the rule where security enabled is default and we only mention "No-Security" when there is no security.

Comment (weicongs-amazon) [reply .github/workflows/test-build-rpm.yml:343]: Currently different workflow files use different naming conventions. I will create a backlog issue for cleanup

Comment (weicongs-amazon) [reply .github/workflows/test-build-rpm.yml:343]: https://github.com/opendistro-for-elasticsearch/opendistro-infra/projects/8

**Pull request merged**